docs/concept-exercises.md

```diff
@@ -93,9 +93,68 @@ See the C# floating-point-numbers exercise's [design.md file][csharp-docs-design
 
 ### `.meta/config.json`
 
-This file contains meta information on the exercise, which currently includes the exercise's contributors, and (optionally) language version requirements.
-
-See the C# floating-point-numbers exercise's [config.json file][csharp-docs-config.json] for an example.
+This file contains meta information on the exercise, which may include
+
+- the exercise's contributors
+- the exercise's authors
+- which exercise it was forked from
+- language version requirements
```

Comment:
Nice! Maybe we can make it more clear what parts are required and which aren't?

```suggestion
This file contains meta information on the exercise:

- The exercise's authors (required)
- The exercise's contributors (optional)
- The exercise it was forked from (optional)
- Language version requirements (optional)
```

Reply:
Same issue as in #1149. Perhaps we could make it more explicit: `(required if the exercise is forked from another)` instead of `(optional)`?

Reply:
Excellent suggestion!